_episodes/03-create.md

```diff
@@ -152,7 +157,7 @@ holding it down, press the O key) to write our data to disk
 (we'll be asked what file we want to save this to:
 press Return to accept the suggested default of `draft.txt`).
 
-![Nano in Action](../fig/nano-screenshot.png)
+<div style="width:80% margin: auto;">![Nano in Action](../fig/nano-screenshot.png)</div>
```

Comment:
I just reverted the change on this line because I noticed the inner markdown wasn't being rendered (i.e. it was showing up as the literal text `![Nano in Action](../fig/nano-screenshot.png)` rather than as an image). 

If you still want to make this change, the simplest thing to do might be to replace the markdown with just an `<img>` element. There may be other workarounds though ([relevant StackOverflow post](https://stackoverflow.com/questions/29368902/how-can-i-wrap-my-markdown-in-an-html-div)).

Reply:
OK. The reason I added that is that the image appeared oversize making it the centerpiece in a lesson where it was supposed to be just a side note. Here is a fully HTML suggestion

```suggestion
<div style="width:80% margin: auto;"><img alt="Nano in Action" src="../fig/nano-screenshot.png"></div>
```

Reply:
[Done](http://swcarpentry.github.io/shell-novice/03-create/index.html). Thanks!